Shop_Unity_base/Assets/Scripts/Shop/States/ShopBrowseState.cs

```diff
@@ -7,48 +7,44 @@ namespace States
     using View;
     using Controller;
 
-    //This state takes the model that is contained in the ModelContainer object, and allow us to browse it using
-    //a controller and two views. Both views are on the same child. On renders the shop as icons, the other renders it
-    //as text (messages). There is no event system, so the text is printed every frame.
+    // This state takes the model that is contained in the ModelContainer object, and allow us to browse it using
+    // a controller and two views. Both views are on the same child. On renders the shop as icons, the other renders it
+    // as text (messages). There is no event system, so the text is printed every frame.
     public class ShopBrowseState : MonoBehaviour
     {
         private ShopModel shopModel;
-
         private ShopController shopController;
+
         private ShopView shopView;
         private ShopMessageView shopMessageView;
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  Start()
-        //------------------------------------------------------------------------------------------------------------------------
-        //This method gets the whole setup going
+        // This method gets the whole setup going
         protected void Start()
         {
-            Initialize();  
+            Initialize();
         }
 
-        //------------------------------------------------------------------------------------------------------------------------
-        //                                                  Initialize()
-        //------------------------------------------------------------------------------------------------------------------------
         public void Initialize()
         {
-            //Set up the model and controller
+            // Set up the model and controller
             shopModel = new ShopModel();
             shopController = new ShopController(shopModel);
 
-            //get view from children
+            // Get view from children
             shopView = GetComponentInChildren<ShopView>();
             Debug.Assert(shopView != null);
 
-            //get mesageview from children
+            // Get mesageview from children
             shopMessageView = GetComponentInChildren<ShopMessageView>();
             Debug.Assert(shopMessageView != null);
 
-            //setup model and controller
+            // Setup model and controller
+            Customer customer = new Customer("Guy");
+            shopModel.SetCustomer(customer);
 
-            //link them
-            shopView.Initialize(shopModel, shopController);//view1
-            shopMessageView.Initialize(shopModel);//view2
+            // Link them (view1 and view2)
```

Comment:
Would suggest removing "(view1 and view2)". It's obvious which views they are (because they are both views), and the number denominators don't really serve any purpose here since the comments no longer specifically denominate them as they did previously.